NameError: uninitialized constant Rpush::Fcm (NameError)

**Describe the bug**
Hello everyone, we are migrating from Gcm to Fcm after getting this error message: 

> [ERROR] Rpush::DeliveryError, Unable to deliver notification 19929, received error (Failed to deliver to all recipients. Errors: DeprecatedApi.)

But after using Rpush::Fcm::App.new we are getting en error: 

> NameError: uninitialized constant Rpush::Fcm (NameError)
>     fcm_app = Rpush::Fcm::App.new
> Did you mean?  Rpush::Gcm

**To Reproduce**
Steps to reproduce the behavior:
1. Snipped code of the rake task: 
```
    fcm_app = Rpush::Fcm::App.new
    fcm_app.name = "android"
    fcm_app.firebase_project_id = ENV['FIREBASE_PROJECT']
    fcm_app.json_key = ENV['RPUSH_FCM_JSON_KEY']
    fcm_app.connections = 30
    fcm_app.save!
```
2. Run the rake task
3. Get error:
> NameError: uninitialized constant Rpush::Fcm (NameError)
> 
>     fcm_app = Rpush::Fcm::App.new
>                    ^^^^^
> Did you mean?  Rpush::Gcm


**Expected behavior**
We expected to not get this error, since this Rpush::Fcm::App.new is mentioned into the README section of Rpush gem.

**Additional context**
It is quite urgent for us to address this problem because we don't have the time to re write the code using another solution.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -12,7 +12,21 @@
 
 * Dropped support for Ruby 2.4, 2.5, 2.6 and Rails 5.2.
 
-[Full Changelog](https://github.com/rpush/rpush/compare/v7.0.1...HEAD)
+[Full Changelog](https://github.com/rpush/rpush/compare/v8.0.0...HEAD)
+
+## [v7.0.1](https://github.com/rpush/rpush/tree/v8.0.0) (2024-09-06)
+
+**Merged pull requests:**
+
+* Support for FCMv1 [\#620](https://github.com/rpush/rpush/pull/620) ([mirkode](https://github.com/mirkode)), [\#660](https://github.com/rpush/rpush/pull/660) ([AnilRh](https://github.com/AnilRh)) and [\#673](https://github.com/rpush/rpush/pull/673) ([SixiS](https://github.com/SixiS), [Henridv](https://github.com/Henridv) & [benlangfeld](https://github.com/benlangfeld))
+* No longer silence content-available notifications for APNs. Reverts the following change from the v6.0.0 release. See https://github.com/rpush/rpush/issues/647.
+  * Fix silent APNS notifications for Apns2 and Apnsp8 [\#596](https://github.com/rpush/rpush/pull/596) ([shved270189](https://github.com/shved270189))
+
+**Breaking:**
+
+* Dropped support for Ruby 2.4, 2.5, 2.6 and Rails 5.2.
+
+[Full Changelog](https://github.com/rpush/rpush/compare/v7.0.1...v8.0.0)
 
 ## [v7.0.1](https://github.com/rpush/rpush/tree/v7.0.1) (2022-03-02)
 

--- a/lib/rpush/version.rb
+++ b/lib/rpush/version.rb
@@ -1,8 +1,8 @@
 module Rpush
   module VERSION
-    MAJOR = 7
+    MAJOR = 8
     MINOR = 0
-    TINY = 1
+    TINY = 0
     PRE = nil
 
     STRING = [MAJOR, MINOR, TINY, PRE].compact.join(".").freeze

